src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Why pass the string "day" as an argument? Also, what is contained in the `day` variable exactly? It'd definitely be crucial to also get the `eventData`